Formula/sevenzip.rb

```diff
@@ -0,0 +1,36 @@
+class Sevenzip < Formula
+  desc "7-Zip is a file archiver with a high compression ratio"
+  homepage "https://7-zip.org"
+  url "https://7-zip.org/a/7z2102-src.7z"
+  version "21.02a"
+  sha256 "ee9755feaa034e7075712bc6b1fad25440bd0a8f21f106a16e37bc4b409f1122"
+  license all_of: ["LGPL-2.1-or-later", "BSD-3-Clause"]
+
+  on_linux do
+    depends_on "gcc@9" => :build
```

Comment:
Why does this depend specifically on gcc 9? That seems like an odd choice. Also, does it not need the gcc runtime?

Reply:
7zip won't compile with gcc 10, because some new warnings are treated as error(e.g. `-Werror=maybe-uninitialized` ). And it does need the gcc runtime, I'll correct it next release.